src/appleseed.studio/mainwindow/rendering/renderwidget.cpp

```diff
@@ -131,6 +131,13 @@ namespace
     }
 }
 
+void RenderWidget::start_render()
+{
+    // Clear the image storage.
+    if (m_image_storage.get())
```

Comment:
Hm, that's normally already taken care of (we're dimming the previous render, on purpose). Why is this necessary?